client/my-sites/plugins/plugins-browser-item/index.jsx

```diff
@@ -142,11 +135,22 @@ class PluginsBrowserListElement extends Component {
 }
 
 export default compose(
-	connect( ( state ) => {
+	connect( ( state, { currentSites, plugin, site } ) => {
 		const selectedSiteId = getSelectedSiteId( state );
 
+		const sitesWithPlugin =
+			site && currentSites
+				? getSitesWithPlugin(
+						state,
+						// eslint-disable-next-line wpcalypso/redux-no-bound-selectors
```

Comment:
I'm planning to address every instance of this `sites.map( site => site.ID )` in a separate PR.